docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
+	robust python open source community.
 
 ## Opening up a terminal 
-*For Mac devices:*
-Click the Launchpad icon in the Dock, type Terminal in the search field, then click Terminal. In the Finder, open the /Applications/Utilities folder, then double-click Terminal.
-
-*For Windows devices:*
-Press the Start taskbar button. Select All apps on the Start menu. Scroll down the Start menu to the Windows Terminal app or Command Prompt app (Windows Terminal is a little nicer if you have it but not required) shown directly below. Then click Windows Terminal or Command Prompt there to open it.
+- **On Mac:**
+Click the Launchpad icon in the Dock, type Terminal in the search field, then click
+Terminal. In the Finder, open the /Applications/Utilities folder, then double-click
+Terminal.
 
+- **On Windows:**
+Press the Start taskbar button. Select All apps on the Start menu. Scroll down the
+Start menu to the Windows Terminal app or Command Prompt app (Windows Terminal is a
+little nicer if you have it but not required) shown directly below. Then click Windows
+Terminal or Command Prompt there to open it.
 
 ## Setting up environment 
-* Download miniconda from [here](https://docs.conda.io/en/latest/miniconda.html) (click the first Linux link)
-	* If using Windows Subsystem for Linux on Windows, use the Linux installer
-* Install miniconda
-	* Change directories to the folder where you downloaded miniconda
-	* Type `bash`
-	* Type `cd /Downloads` (type ls to see whether the Miniconda is in the working directory, and how to type that folder name)
-	* Type `bash [miniconda file name]` (e.g., Miniconda3) 
-* Answer yes to all the questions 
-	* Tip: press the spacebar to go through the terms and conditions more quickly
-* Clone the repository in the terminal: 
-	* Change directories to your home directory
-		* Type `cd`
-	* Type `git clone https://github.com/Center-for-Women-s-Welfare/SSS.git`
-	* The default directory when you open a terminal is usually under the user, so use cd to find the working directory you are in 
-	* Type `cd ..` to navigate back to the previous working directory in the last terminal session 
-* Change directories into the new folder you created
-	* Type `cd SSS`
-* Create a conda environment for this work
-	* Type `conda env create -f sss.yml`
-* Activate this environment
-	* Type `conda activate sss`
-	* To confirm you are in the sss environment, the parentheses at the very beginning of the line should say (sss)
-* Install the sss package
-	* Type `pip install .`
-	* *Dot means install the current directory package. So, it’s important that you are in the sss directory*
-
-## Changing your directory to sss
-Change the directory into the working directory that contains the repository you just cloned:
-* Type `cd SSS`
+- Download miniconda from [here](https://docs.conda.io/en/latest/miniconda.html)
+	- If using Windows Subsystem for Linux on Windows, use the Linux installer
```

Comment:
This is the only place WSL is mentioned in this document. Is there a good way to tell the reader whether or not they need to worry about this bullet point? Fwiw these are the outputs of `$ uname` for me in WSL: `Linux` and in Git Bash: `MINGW64_NT-10.0-22000`